### PR TITLE
Show cursor shape in SUI

### PR DESCRIPTION
**Pull request by PankajBhojwani**
Show the cursor shape next to the text for it

References #8764 

cursorshape

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -497,19 +497,19 @@
...
-    <value>Filled box</value>
+    <value>Filled box ( █ )</value>
   </data>
   <data name="Profile_CursorShapeUnderscore.Content" xml:space="preserve">
-    <value>Underscore</value>
+    <value>Underscore ( ▁ )</value>
   </data>
   <data name="Profile_CursorShapeVintage.Content" xml:space="preserve">
-    <value>Vintage</value>
+    <value>Vintage ( ▃ )</value>
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:512]: Add a resw comment for each of these to lock the `(<example>)` text.

**Comment by mdtauk** [comment]
If you implement the console preview display, this too could display the chosen cursor shape, colour, and blink rate (whatever settings you include for the cursor)

**Review by carlos-zamora — APPROVED** [review]

**Comment by carlos-zamora** [comment]
> If you implement the console preview display, this too could display the chosen cursor shape, colour, and blink rate (whatever settings you include for the cursor)

Yup! Unfortunately, I don't think we have enough time to add the terminal preview for v1.6. But it's a very exciting feature for v1.7 😀

**Comment by zadjii-msft** [comment]
I'm kinda shocked that these blocks are different widths. I would've figured they're the same width, one _narrow_ character. 


Oh maybe if you make the font for just the run with the cursor a monospaced font, they'd be narrow?

**Comment by DHowett** [comment]
That would require there to be multiple Runs eh

**Review by DHowett — APPROVED** [review]
I'm comfortable with this; we can do better later. Sucks that it requires another localization pass.

**Comment by DHowett** [comment]
@msftbot merge this in 12 hours

**Review by zadjii-msft — APPROVED** [review]

**Pull request merged**